assets/src/components/ColorPaletteControl/ColorPaletteControl.js

```diff
@@ -7,13 +7,15 @@ import { ColorPalette } from '@wordpress/components';
 function ColorPaletteControl( { label, className, value, help, instanceId, onChange, options = [], ...passThroughProps } ) {
   const id = `inspector-color-palette-control-${ instanceId }`;
 
+  const optionsAsColors = options.map( option => ( { color: option.value, name: option.label } ) );
```

Comment:
Maybe an easier solution here is to spread so that we don't need to re-specify the `name` key and are future proof for other fields.
```
const { value, ...otherFields  } = option;
return {
    color: value,
    ...otherFields,
}
```

Reply:
Resolving this as we decided to remove the labels from the JSON files.